css-grid-2/Overview.bs

```diff
@@ -1593,11 +1593,11 @@ Explicit Track Sizing: the 'grid-template-rows' and 'grid-template-columns' prop
 		<dfn>&lt;explicit-track-list></dfn> = [ <<line-names>>? <<track-size>> ]+ <<line-names>>?
 
 		<dfn>&lt;line-name-list></dfn>      = [ <<line-names>> | <<name-repeat>> ]+
-		<dfn>&lt;track-size></dfn>          = <<track-breadth>> | minmax( <<inflexible-breadth>> , <<track-breadth>> ) | fit-content( <<length-percentage>> )
+		<dfn>&lt;track-size></dfn>          = <<track-breadth>> | minmax( <<inflexible-breadth>> , <<track-breadth>> ) | fit-content( <<length-percentage [0,∞]>> )
```

Comment:
I am not sure about this one because [the related paragraph in the spec](https://drafts.csswg.org/css-grid-2/#valdef-grid-template-columns-fit-content) does not event define a negative value as invalid.